src/generator.ts

```diff
@@ -258,12 +258,12 @@ export class Generator {
     let gitIgnore = '';
     if (fs.existsSync(gitIgnorePath))
       gitIgnore = fs.readFileSync(gitIgnorePath, 'utf-8').trimEnd() + '\n';
-    if (!gitIgnore.includes('node_modules'))
-      gitIgnore += 'node_modules/\n';
-    gitIgnore += '/test-results/\n';
-    gitIgnore += '/playwright-report/\n';
-    gitIgnore += '/blob-report/\n';
-    gitIgnore += '/playwright/.cache/\n';
+    const valuesToAdd = ['node_modules/', '/test-results/', '/playwright-report/', '/blob-report/', '/playwright/.cache/'];
+    valuesToAdd.forEach(value => {
+      if (!gitIgnore.includes(value)) {
```

Comment:
the `.includes()` call here would cause a false positive on lines that include the paths:

```
/some-library/test-results/
```

this would then lead to those affected Playwright paths not being added to `.gitignore`

so probably a more strict matching should be done here (eg. splitting `gitIgnore` by lines)

Reply:
true, I did not think of that. I personally would prefer the approach of using regex to check in this case, this way it wouldn't be necessary to create a new array and looping over two arrays instead of one